@xen-orchestra/rest-api/src/index.mts

```diff
@@ -0,0 +1,11 @@
+import * as swaggerUi from 'swagger-ui-express'
+import type { Express } from 'express'
+
+import swaggerOpenApiSpec from './open-api/spec/swagger.json' assert { type: 'json' }
```

Comment:
This fails with newer version of Node as it seems that `assert` has been dropped in favor of `with`

https://github.com/nodejs/node/issues/51622

Reply:
this may be tricky since we are running node 20  and I am not sure it supports the with syntax ( thank you for using unsearchable terms to name feature .... )